Formula/box2d.rb

```diff
@@ -0,0 +1,30 @@
+class Box2d < Formula
+  desc "2D physics engine for games"
+  homepage "https://box2d.org"
+  url "https://github.com/erincatto/box2d/archive/v2.4.0.tar.gz"
+  sha256 "38bce8217afb974e5be070da0b6768bb4fc0e246fa1cee42fecf71c4c6e9ad19"
+  license "MIT"
+  head "https://github.com/erincatto/Box2D.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = std_cmake_args + %w[
+      -DBOX2D_BUILD_UNIT_TESTS=OFF
+      -DBOX2D_BUILD_TESTBED=OFF
+      -DBOX2D_BUILD_EXAMPLES=OFF
+    ]
+
+    system "cmake", ".", *args
+    system "cmake", "--build", "."
+    include.install Dir["include/*"]
+    lib.install "src/libbox2d.a"
+    libexec.install "unit-test/hello_world.cpp", "unit-test/doctest.h"
+  end
+
+  test do
+    system ENV.cxx, "-L#{lib}", "-lbox2d", "-std=c++11",
+      libexec/"hello_world.cpp", "-o", testpath/"test"
+    system "./test"
```

Comment:
Can we check the output?

Reply:
It returns exit code other than 0 when failing making brew test report the test as failed (and dumping the output IIRC).

Reply:
Can we still check the output?

Reply:
Sure.

Reply:
It's like @SMillerDev reads my mind

Reply:
Done, I've pushed the changes since the issues in box2d repository do not get much attention, I will update the formula when the `cmake --install` is available.